### DISCUSSION TITLE
Pro Tools 2020.12.0 - Issues with Network Servers

### DISCUSSION THREAD

**Discussion by MistaMin** [apohl79/audiogridder#441] [Help]
Hi! 

Currently I am having issues with Audiogridder on my 12 Core (2x 6 core 3.06 gHz) Mid 2010 Mac Pro, running High Sierra, in Pro Tools 2020 when running Audiogridder Server on a second computer (Windows 10 with latest updates 8 Core Xeon based HP Z600 workstation).
With a buffer size of 8 blocks I keep getting random System Usage spikes with no plugins except 1 instance of Audiogridder and 1 other native plugin.
 
When I run local server I can fit more instances, but also run into cpu usage problems before even reaching 40+ % usage.

I'm guessing the problem is with pro tools since with studio one I didn't have the same issues, though I don't use studio one to mix in and used a demo to test the system.

I have the latest version of audiogridder installed 1.1.1. running as an AAX plugin.

Please let me know what kind of screenshots I should take and what information is necessary to help figure how to get this to work.

Love the plugin and it's already helped me get a song mixed via local server.

**Comment by MistaMin** [comment]
So I just read the Bug Report document... I'll try to post those in the next hour.

**Reply by apohl79** [reply]
Thanks for reporting this. and yes, gathering info as described in the bug reporting guide is the best you can do.

I will spend more time with PT myself. I have not run larger projects on it, so i'm lacking experience with AG on it right now.

**Comment by MistaMin** [comment]
So after trying to run a small test session with just one or two instances of audiogridder, I'm getting the spinning wheel of death in Pro Tools.  I attached the local sever info and my windows server. It's still the same scenario as mentioned above, audiogridder does not want to work propperly when there is more than one instance of it or any other plugin.

There was no crash report because pro tools didn't actually crash, but instead froze.

Also, the /cores/core.* file mentioned in the log was not to be found in the folder.

Thanks so much for taking time to check this out!
[Local Server -AudioGridder.zip](https://github.com/apohl79/audiogridder/files/6167580/Local.Server.-AudioGridder.zip)

[Windows Server - AudioGridder.zip](https://github.com/apohl79/audiogridder/files/6167582/Windows.Server.-.AudioGridder.zip)

**Comment by LoZio80** [comment]
https://user-images.githubusercontent.com/80973200/111808134-00098400-88d4-11eb-9f4e-cf305f2019b2.mov

**Reply by MistaMin** [reply]
This is exactly what happens to me. No CPU usage and then random peaks.

Kind of disheartening really 😔

**Comment by MistaMin** [comment]
Did some testing again, but now in Studio One and it was more stable, but continued to have dropout and changing the Buffer in audiogridder didn't help. I don't know what to do concerning system stability and would love to have some feedback from any user who could let me know what I could do to better the situation.

In Studio One, I was running version 5.2 with two plugins open and it crashed while switching from my Windows Sever to a local server. On the windows sever there was no stability at all in the session and drop out continued to occur.

[WIndows Sever AudioGridder 210320.zip](https://github.com/apohl79/audiogridder/files/6175304/WIndows.Sever.AudioGridder.210320.zip)
[Local Server - AudioGridder 210320.zip](https://github.com/apohl79/audiogridder/files/6175305/Local.Server.-.AudioGridder.210320.zip)
[Mac Pro Crashes 210320.zip](https://github.com/apohl79/audiogridder/files/6175306/Mac.Pro.Crashes.210320.zip)
[Old Crashes - Archive.zip](https://github.com/apohl79/audiogridder/files/6175307/Old.Crashes.-.Archive.zip)

**Reply by UncoveredPitch** [reply]
I'm new to Audiogridder as well but if you like have a look at "my" thread. Our computers probably have similar power but I do use Reaper which is supposedly better at running plug-ins than ProTools or Studio One. If you picture your server as a dog on a lead, keep the lead very short and don't twist it. Meaning keep the network itself as clean and fast as possible, with no other apps like a browser or email client running, and keeping the buffer size in Audiogridder to 1 or 2 blocks. Also, if your DAW allows you to compute plug-in FX in advance at the cost of additional latency, keep that pre-render time as short as possible. 

Let me assure you I tried for days(sometimes in desperation with really high buffer settings) until things clicked into place, and I still don't exactly know why. In my video you can see how CPU usage spikes from next to zero to much more than the plug-in would have used natively. But like I said, making the whole system as fast as possible was an important factor in how it works now. 0.01% CPU for each instance of Audiogridder on the client and predictable CPU usage on the server to about 70%, as you need to keep the rest in reserve for opening the UI and making tweaks via remote screen control. Good luck!

**Reply by MistaMin** [reply]
YOU SIR ARE A GENIUS!!!!! I connected the other workstation directly to the Mac Pro via the second ethernet port and setup internet sharing in mac OS and now via this "mini"-network I have no issues what so ever!!!! AMAZING!!!!!!!!!!!

I did switch back to 1.1.0 due to some other issues with local server, but this is just great!!!!!

**Reply by UncoveredPitch** [reply]
Good stuff. I think I was lucky overall with my own rig but I'm glad that my dog walking analogy got you closer to a well-functioning setup!

**Comment by CaBuffalo** [comment]
It's so great to see it's all worked out, thanks for sharing the
information.

Enjoy the AG !! :)

Cheers.

On Tue, Mar 23, 2021 at 11:35 AM MistaMin ***@***.***> wrote:

> YOU SIR ARE A GENIUS!!!!! I connected the other workstation directly to
> the Mac Pro via the second ethernet port and setup internet sharing in mac
> OS and now via this "mini"-network I have no issues what so ever!!!!
> AMAZING!!!!!!!!!!!
>
> I did switch back to 1.1.0 due to some other issues with local server, but
> this is just great!!!!!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/441#discussioncomment-519239>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BFUOWFZYNCVW333WTTFDNNLANCNFSM4ZNO2VEA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by MistaMin** [reply]
Well, it's still not perfect, but atleast it's stable! :)
